packages/App/src/Navigation.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {NavigationContainer} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import Balance from './Balance';
+import LogIn from './Registaration/LogIn';
+import SignUp from './Registaration/SignUp';
+import Loading from './components/Loading';
+import {userIsLogIn} from './store/selectors/userIsLoged';
+import {IsLoadingUser} from './store/selectors/userIsLoading';
+import {useSelector} from 'react-redux';
+
+const Stack = createNativeStackNavigator();
+
+const Navigation = () => {
+  const isLogIn = useSelector(state => userIsLogIn(state));
```

Comment:
You're creating a new selector function on every render. You can just pass selector by reference